langmeta/langmeta/shared/src/main/scala/org/langmeta/internal/inputs/package.scala

```diff
@@ -25,27 +25,16 @@ package object inputs {
     }
   }
 
-  // TODO: the extension methods below are temporary stubs that should be moved to the public API
-
+  
+  @deprecated("Use Input.syntax or Input.strucure", "2.1.4")
```

Comment:
FYI @xeno-by This is deprecated because these methods leaked into the public api without internal imports

```scala

@ import $ivy.`org.scalameta::contrib:2.0.0`, scala.meta._, contrib._
import $ivy.$                             , scala.meta._, contrib._

@ Input.VirtualFile("a.scala", "object A").syntax
res1: String = "a.scala"

```

Reply:
Wow. Got it.